### PR TITLE
Documentation for default roles

### PR DESCRIPTION
**Pull request by aleksandrychev**

### CODE REVIEW

**Comment by aleksandrychev** [comment]
@nickanderson @dmitry84 Plese review

File: enterprise-cfengine-guide/settings.markdown [hunk 1]
```diff
@@ -66,12 +66,26 @@ will not see any hosts in Mission Portal.
 Users without a role will not be able to see any hosts in Mission
 Portal.
 
-**Default Roles:**
+**Preset Roles:**
```
[the remark this answers is not in the document]
Comment (nickanderson) [reply enterprise-cfengine-guide/settings.markdown]: Oh i see. I would call these "Predefined Roles"

File: enterprise-cfengine-guide/settings.markdown [hunk 2]
```diff
@@ -66,12 +66,26 @@ will not see any hosts in Mission Portal.
...
 
 * admin - The admin role can see everything and do anything.
 * cf_remoteagent - This role allows execution of cf-runagent. It can
   be used from within Design Center to troubleshoot hosts that have
   failed sketch activations.
+  
+**Default Role:**
+
+On `Role Management` page you can set default role by clicking on checkbox located
```
Comment (nickanderson) [inline enterprise-cfengine-guide/settings.markdown]: @michaelclelland might have some wording suggestions.

I might say "To set a default role navigate to the `Role Management` page and checking the boxes in the `Default role` column which you would like to apply to all new users."

**Comment by aleksandrychev** [comment]
@nickanderson  I've applied your changes.

File: enterprise-cfengine-guide/settings.markdown [hunk 3]
```diff
@@ -66,12 +66,25 @@ will not see any hosts in Mission Portal.
...
+**Predefined Roles:**
 
 * admin - The admin role can see everything and do anything.
 * cf_remoteagent - This role allows execution of cf-runagent. It can
   be used from within Design Center to troubleshoot hosts that have
   failed sketch activations.
+  
+**Default Role:**
+
+To set a default role navigate to the Role Management page and checking the boxes in the Default role column which you would like to apply to all new users.
```
Comment (estenberg) [inline enterprise-cfengine-guide/settings.markdown]: Suggesting: "To set the default role, click Settings -> User management -> Roles. You can then select which role will be the default role for new users.

File: enterprise-cfengine-guide/settings.markdown [hunk 4]
```diff
@@ -66,12 +66,25 @@ will not see any hosts in Mission Portal.
...
+
+![DefaultRoleSelecting](roles-list.png)
+
+**Behaviour of Default Role:**
+
+Once you set up default role, all new users registered through LDAP would have this role.
+When you add new user, the default role will be already selected.
```
Comment (estenberg) [inline enterprise-cfengine-guide/settings.markdown]: Suggesting:
Any new users created in Mission Portal's local user database will have this new role assigned.

Users authenticating through LDAP (if you have LDAP configured in Mission Portal) will get this new role applied the first time they log in.

Note that the default role will not have any effect on users that already exist (in Mission Portal's local database) or have already logged in (when using LDAP).

Comment (estenberg) [reply enterprise-cfengine-guide/settings.markdown]: In effect this allows you to set the default permissions for new users (e.g. which hosts a user is allowed to see) by configuring the access for the default role.

**Comment by nickanderson** [comment]
@aleksandrychev  can you please make the changes suggested by @estenberg

**Comment by aleksandrychev** [comment]
Sure! I am going to finish with documentations on the next Monday.

File: enterprise-cfengine-guide/settings.markdown (hunk 4 above)
Comment (aleksandrychev) [reply enterprise-cfengine-guide/settings.markdown]: I've added your suggestions, thanks!

**Comment by aleksandrychev** [comment]
>@aleksandrychev can you please make the changes suggested by @estenberg

Done

**Pull request merged**